modules/ROOT/pages/add-api-alert-task.adoc

```diff
@@ -1,21 +1,27 @@
-= Adding an API Alert
+= Add API Alerts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-. In API Manager 2.x, in API Administration, select an API instance, and on the API instance dashboard, select Alerts.
-. Click Add Alert and set the alert options:
-* Name: Type an alert name, for example `red-alert`.
-* Enabled: Accept the default, Enabled, for example.
-* Severity: Accept the default, Critical.
-* Alert type: Select Request Count, for example.
-* When number of occurrences are: Select Greater.
-* than: Select 1.
-* For at least: Select an entry in the range 1 - 999999999 to specify the time intervals as a sliding windows.
-* Recipients: Select the names of users in your business group, or type the email addresses, of users who will receive notification when the alert fires. Press RETURN.
+You can add one or more API alerts for your API instances to track and manage APIs.
```

Comment:
"add to" rather than "add for"
```suggestion
You can add one or more API alerts to your API instances to track and manage APIs.
```
BETTER
```suggestion
To track and manage API performance, add one or more alerts to your API instances.
```

Reply:
Done! Thanks!